EIPS/eip-2357.md

```diff
@@ -0,0 +1,37 @@
+---
+eip: 2357
+title: Total Difficulty in Block Header
+author: Tomasz K. Stanczak (@tkstanczak)
+discussions-to: https://ethereum-magicians.org/t/eip-2357-replace-difficulty-with-total-difficulty-in-block-headers/4233
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-04-29
+---
+
+## Simple Summary
+
+This document proposes to replace Dififculty with Total Difficulty in the Ethereum block headers..
+
+## Abstract
+
+Replace field Difficulty with TotalDifficulty (sum of difficulties since genesis) in the Ethereum block header.
+
+## Motivation
+
+Simplify total difficulty verification in stateless clients.
+
+## Specification
+
+Difficulty field will be replaced with TotalDifficulty (sum of difficulties since genesis) in the Ethereum block header.
+
+## Rationale
+
+This simplifies the selection of the best peer, acceptance of the NewBlock broadcasts and stateless client execution without verification from genesis.
+
+## Backwards Compatibility
+
+Clients will have to understand that headers before the EIP will have Difficulty and not total difficulty field.
```

Comment:
I suggest to consistently use `TotalDifficulty`, ` Total Difficulty` or `total difficulty`.